packages/core/src/singleton/Storage/types.ts

```diff
@@ -6,6 +6,13 @@ import { AtLeastOne } from '../types';
 /** @deprecated This may be removed in the next major version. */
 export type StorageAccessLevel = 'guest' | 'protected' | 'private';
 
+/** Information on bucket used to store files/objects */
+export interface BucketInfo {
+	/** Actual bucket name */
+	bucketName: string;
```

Comment:
If the interface is `BucketInfo`, then the `bucket` prefix  in `bucketName` is redundant. 

```suggestion
name: string
```

Reply:
This is intended. `bucketName` is how backend refers to it and `name` is used to refer to friendly name. With gen2 work, we decided to align with backend whenever possible so customers will find it easy to map